devel/ansible-podman/roles/podman/tasks/start.yml

```diff
@@ -0,0 +1,94 @@
+- name: Create the pod
+  containers.podman.podman_pod:
+    name: bodhi-dev
+    state: started
+    publish:
+      - "6543:6543"
+      - "6544:6544"
+      - "6545:6545"
+      - "6546:6546"
+      - "15672:15672"
+
+- name: Run the database container
+  containers.podman.podman_container:
+    name: bodhi-dev-database
+    image: "docker.io/library/postgres:latest"
+    env:
+      POSTGRES_HOST_AUTH_METHOD: "trust"
+    state: started
+    expose:
+      - 5432
+    pod: bodhi-dev
+    volume:
+      - /tmp/bodhi-dev/init-db.sh:/docker-entrypoint-initdb.d/init-db.sh:Z
+      - /tmp/bodhi-dev/waiverdb.dump.xz:/docker-entrypoint-initdb.d/waiverdb.dump.xz:Z
+      - /tmp/bodhi-dev/bodhi2.dump.xz:/docker-entrypoint-initdb.d/bodhi2.dump.xz:Z
+    healthcheck: "test -f /tmp/.init-done && pg_isready -h localhost -U waiverdb -d waiverdb"
+
+- name: Run the waiverdb container
+  containers.podman.podman_container:
+    name: bodhi-dev-waiverdb
+    image: "quay.io/factory2/waiverdb:latest"
+    state: started
+    pod: bodhi-dev
+    volume:
+      - /tmp/bodhi-dev/waiverdb-settings.py:/etc/waiverdb/settings.py:Z
+      - /tmp/bodhi-dev/run_waiverdb.sh:/usr/local/bin/run_waiverdb.sh:Z
+    healthcheck: "python3 -c \"import urllib.request; urllib.request.urlopen('http://localhost:6544/healthcheck')\""
+    entrypoint: "/usr/local/bin/run_waiverdb.sh"
+
+- name: Run the greenwave container
+  containers.podman.podman_container:
+    name: bodhi-dev-greenwave
+    image: "quay.io/factory2/greenwave:latest"
+    state: started
+    pod: bodhi-dev
+    volume:
+      - /tmp/bodhi-dev/greenwave-settings.py:/etc/greenwave/settings.py:Z
+      - /tmp/bodhi-dev/fedora.yaml:/etc/greenwave/policies/fedora.yaml:Z
+    healthcheck: "python3 -c \"import urllib.request; urllib.request.urlopen(http://localhost:6545/api/v1.0/about')\""
```

Comment:
```suggestion
    healthcheck: >-
    python3 -c "import urllib.request; urllib.request.urlopen('http://localhost:6545/api/v1.0/about')"
```

Reply:
ha, that floating single quote is probably why this container keeps showing up as unhealthy...

Reply:
what does your suggestion here mean, exactly, though? the `>` makes it a folded block, but we're only using one line, so...why? and what does the dash mean?

Reply:
The `-` strips the trailing newline from the folded block. I suggested this approach so you don't have to deal with escaping quotes.

Reply:
ah, OK. Meh, honestly, for me personally the escaped quotes are more easily understandable than the folded block magic :) Mind if we leave it as-is? This was profitable anyway, though, as it made me find that missing quote...

Reply:
Fine with me!